.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "openpilot devcontainer",
+  "initializeCommand": "docker build -t openpilot_base -f Dockerfile.openpilot_base .",
```

Comment:
This isn't great, since it needs the whole context of the openpilot folder to build (quite slow).
Ideally it just takes in the files it needs.

We could also base the dev container on the one in the image repo, but that wouldn't take into account local changes